src/Microsoft.Android.Sdk.ILLink/Microsoft.Android.Sdk.ILLink.csproj

```diff
@@ -1,17 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\Configuration.props" />
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
silly question, but does this need to be net6.0? or can it be netstandard2.1?

Reply:
it needs to be at least net5.0 IIRC

Reply:
```
src\Microsoft.Android.Sdk.ILLink\Microsoft.Android.Sdk.ILLink.csproj : error NU1202: Package Microsoft.NET.ILLink 6.0.0-alpha.1.211
09.1 is not compatible with netstandard2.1 (.NETStandard,Version=v2.1). Package Microsoft.NET.ILLink 6.0.0-alpha.1.21109.1 supports: net5.0 (.NETCoreApp,Version=v
5.0)
```

Reply:
ok , was just asking :)